Pidaq/Lib/database/database.py

```diff
@@ -0,0 +1,194 @@
+            measurements (list(str)): Contains the measurements to be queried
+
+        Returns:
+            (str) Raw json data
+        """
+
+        if tags is not None and measurements is None:
+            formatted_tags = format_data(tags)
+            data = self.client.query(
+                f'SELECT {formatted_tags}',
+                database=self.current_database
+            )
+        elif tags is None and measurements is not None:
+            formatted_measurements = format_data(measurements)
+            data = self.client.query(
+                f'SELECT * FROM {formatted_measurements}',
+                database=self.current_database
+            )
+        elif tags is not None and measurements is not None:
+            formatted_tags = format_data(tags)
+            formatted_measurements = format_data(measurements)
+            data = self.client.query(
+                f'SELECT {formatted_tags} FROM {formatted_measurements}',
+                database=self.current_database
+            )
+        else:
+            return -1 # Must specify at least a tag or a measurement
+        try:
+            return data.raw['series']
+        except KeyError:
+            return -1
+
+    def create_retention_policy(self, name: str, duration: str, replication: str) -> None:
+        """Creates a retention policy for the current database"""
+
+        self.client.create_retention_policy(
+            name,
+            duration,
+            replication,
+            database=self.current_database,
+            default=True
+        )
+
+    def export_to_csv(self, test_name: str, tags=None, measurements=None, csv_path=None) -> None:
+        """Exports measurements to a csv file
+
+        Args:
+            test_name (str): Name of test, becomes name of csv file
+            tags (list(str)): Tags and fields to be included in csv
+            measurements (list(str)): Measurements to be included in csv
+            csv_path(str): File path of where the csv will be written
+        """
+
+        file_name = f'{test_name}.csv'
+        data = self.read_data(tags=tags, measurements=measurements)
+        if csv_path is None:
+            csv_path = os.getcwd()
+        with open(f'{csv_path}\\{file_name}', 'w', newline='\n') as csvfile:
+            writer = csv.writer(csvfile, delimiter=',')
+            writer.writerow(data[0]['columns'])
+            for measurement in data:
+                for row in measurement['values']:
+                    writer.writerow(row)
+
+
+def format_data(data: list) -> str:
+    """Formats data to be able to utilize in the Influx query
+
+    Args:
+        data(list(str)): List of data to be formatted
+
+    Returns formatted_data(str): comma separated data in format
+                                 '"data1","data2","data3"...'
+    """
+
+    list_data = []
+    for point in data:
+        list_data.append(f'"{point}"')
+    formatted_data = ','.join(list_data)
+    return formatted_data
```

Comment:
Does this function do [1,2,3] -> ' "1", "2" , "3", ' ?

If so, there is a one line solution (not sure it needs to be a function, it is cleaner in a function though admittedly:

```
','.join([f'"{num}"' for num in lst])
```

Reply:
that'll work, gotta love a good list comprehension